pkgs/development/interpreters/python/hooks/conda-install-hook.sh

```diff
@@ -0,0 +1,32 @@
+# Setup hook to use in case a conda binary package is installed
+echo "Sourcing conda install hook"
+
+condaInstallPhase(){
+    echo "Executing condaInstallPhase"
+    runHook preInstall
+
+    # There are two different formats of conda packages.
+    # It either contains only a site-packages directory
+    # or multiple top level directories.
+    siteDir=@pythonSitePackages@
+    if [ -e ./site-packages ]; then
+      mkdir -p $out/$siteDir
+      cp -r ./site-packages/* $out/$siteDir
+    else
+      cp -r . $out
+      rm $out/env-vars
+    fi
+
+    # remove condas place holder for the interpreter path.
+    if [ -e $out/bin ]; then
+      find $out/bin -type f -exec sed -i "s|/opt/anaconda1anaconda2anaconda3||g" {} \;
```

Comment:
You are sure you are not missing a separator here?

Reply:
Separator at what place? It is a while ago when I wrote this. I will try to find a conda package which is affected by this and test it again. The reason I had to remove the `/opt/anaconda1anaconda2anaconda3` prefix from shebangs might be because `patchPythonScript`wouldn't work correctly otherwise. Does that make sense? Anyways, I will check it again.

Reply:
Some python scripts in conda packages start with this line:
`#!/opt/anaconda1anaconda2anaconda3/bin/python`

Reply:
I removed the replacement now. I tested it and shebangs were patched correctly anyways.

Reply:
> Separator at what place?

``/opt/anaconda1anaconda2anaconda3`` looks very strange. I thought it would need to be ``/opt/anaconda1,/opt/anaconda2,/opt/anaconda3``